CollapseLauncher/XAMLs/MainApp/Pages/RepairPage.xaml.cs

```diff
@@ -175,7 +175,8 @@ private void _repairTool_ProgressChanged(object sender, TotalPerfileProgress e)
                 double percentage = double.IsInfinity(e.ProgressPerFilePercentage) ? 0 : e.ProgressPerFilePercentage;
                 RepairPerFileProgressBar.Value = percentage;
 
-                RepairTotalProgressBar.Value = e.ProgressTotalPercentage;
+                double totalPercentage = double.IsInfinity(e.ProgressTotalPercentage) ? 0 : e.ProgressTotalPercentage;
+                RepairTotalProgressBar.Value = totalPercentage;
```

Comment:
We can just do `RepairTotalProgressBar.Value = double.IsInfinity(e.ProgressTotalPercentage) ? 0 : e.ProgressTotalPercentage;` and save a line that way, but it's optional since the previous lines do it that way anyway.

Reply:
I think assigning the value to a variable will be reliable enough to debug where the values have fallen. Furthermore, I suggest implementing the checks around the ``ProgressBase`` class's events so we can prevent the same scenario happening in different cases.